sdk/python/pulumi_azuread/user.py

```diff
@@ -1493,10 +1493,10 @@ def __init__(__self__,
         import pulumi_azuread as azuread
 
         example = azuread.User("example",
+            user_principal_name="jdoe@example.com",
```

Comment:
There's quite a few of these reorders - do you know what caused them? Will this happen only once or is the order no longer stable?

Reply:
The order is stable. It should only change once.